src/api-mocks/appointments.ts

```diff
@@ -71,11 +71,17 @@ interface LocationConfig {
 const LOCATIONS: LocationConfig[] = [
   {
     products: ['166a5c79', 'e8e045ab'],
-    location: {identifier: '1396f17c', name: 'Open Gem'},
+    location: {identifier: '1396f17c', name: 'Open Gem', city: '', address: '', postalcode: ''},
```

Comment:
We have 2 locations with empty strings as well, so these are shown in the stories. If you think we do need separate stories let me know.